datahub-web-react/src/app/glossary/GlossaryEntitiesList.tsx

```diff
@@ -1,44 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Empty } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+
 import styled from 'styled-components/macro';
 import { GlossaryNodeFragment } from '../../graphql/fragments.generated';
-import { GlossaryNode, GlossaryTerm } from '../../types.generated';
+import { EntityType, GlossaryNode, GlossaryTerm } from '../../types.generated';
 import { useEntityRegistry } from '../useEntityRegistry';
 import GlossaryEntityItem from './GlossaryEntityItem';
+import StyledButton from '../entity/shared/components/styled/StyledButton';
+import CreateGlossaryEntityModal from '../entity/shared/EntityDropdown/CreateGlossaryEntityModal';
 
 const EntitiesWrapper = styled.div`
     flex: 1;
     overflow: auto;
     padding-bottom: 20px;
 `;
 
+const ButtonContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`;
+
 interface Props {
     nodes: (GlossaryNode | GlossaryNodeFragment)[];
     terms: GlossaryTerm[];
+    refetchForTerms?: () => void;
+    refetchForNodes?: () => void;
```

Comment:
curious why you added these props since we're only passing them in from the BusinessGlossaryPage and we shouldn't need them there because if there's no terms or nodes we don't even render this component, so we won't use them in here.

Reply:
Removed!!